stdlib/runtime/string.gr

```diff
@@ -178,11 +263,9 @@ let rec heapValueToString = (ptr, extraIndents, toplevel) => {
       }
     },
     t when t == Tags._GRAIN_RECORD_HEAP_TAG => {
-      // these are tagged ints
-      let moduleId = WasmI32.load(ptr, 4n) >> 1n
-      let typeId = WasmI32.load(ptr, 8n) >> 1n
       let recordArity = WasmI32.load(ptr, 12n)
-      if (!recordTypeExists(moduleId, typeId) || recordArity == 0n) {
+      let fields = getRecordFieldNames(ptr)
+      if (fields == -1n || recordArity == 0n) {
```

Comment:
What does an arity-0 record actually look like? Why does it exist?

Reply:
It's currently impossible to make one. I just left this in, but we can delete it. It's also fine as a sanity check. (But probably we can just delete it.)

Reply:
Let's delete it! Less code is always nicer